packages/react-core/src/core/types/wallet.ts

```diff
@@ -19,16 +16,16 @@ export type WalletOptions = {
   dappMetadata: DAppMetaData;
 } & ExtraCoreWalletOptions;
 
-export type SupportedWalletInstance = InstanceType<
-  typeof AbstractBrowserWallet
-> & {
-  connector?: TWConnector;
+export type WalletInstance = InstanceType<typeof AbstractBrowserWallet>;
+
+export type WalletClass<INS extends WalletInstance = WalletInstance> = {
```

Comment:
Can we rename INS to `T` or `I` to conform with generic types being one letter?

Reply:
nah please use descriptive names for generics, the convention is `T<DescriptiveName>` - if ur only using it within one line just `T` is okay too